api/provider/aws/task_create.go

```diff
@@ -1,7 +1,88 @@
 package aws
 
-import "github.com/quintilesims/layer0/common/models"
+import (
+	"fmt"
 
-func (t *TaskProvider) Create(req models.CreateTaskRequest) (*models.Task, error) {
-	return nil, nil
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (t *TaskProvider) Create(req models.CreateTaskRequest) (string, error) {
+	taskID := generateEntityID(req.TaskName)
+	fqEnvironmentID := addLayer0Prefix(t.Config.Instance(), req.EnvironmentID)
+
+	clusterName := fqEnvironmentID
+	startedBy := t.Config.Instance()
+	// todo: deployName, deployVersion, err := lookupDeployNameAndVersion(...)
+
+	taskDefinitionFamily := "l0-zpatrick-test"
```

Comment:
I assume `taskDefinitionFamily` and `taskDefinitionVersion` here are just placeholders for future function calls?

Reply:
fwiw: this is pulled from my working branch. will be fixed in future pr

Reply:
That is my assumption as well. I'm keeping an eye at the moment on progress being made on the task-refactor branch for what will come to replace these.